pages/docs.html

```diff
@@ -77,29 +77,209 @@ <h3>Components</h3>
+              style="
+                width: 100%;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+          <div class="component-type m-t-4">
+            <h3 class="m-b-2">Example of Text Avatar</h3>
+            <p class="m-b-2">
+              Include class
+              <span class="primary-color-text">avatar-text</span>
+            </p>
+          </div>
+          <div class="component-display m-b-2 m-r-4">
+            <div class="avatar avatar-text avatar-lg">RM</div>
+            <div class="avatar avatar-text avatar-md">UI</div>
+            <div class="avatar avatar-text avatar-sm">YZ</div>
+            <div class="avatar avatar-text avatar-xs">CD</div>
+          </div>
+          <div class="component-code m-r-4">
+            <iframe
+              src="https://carbon.now.sh/embed?bg=rgba%28199%2C198%2C247%2C1%29&t=vscode&wt=none&l=htmlmixed&width=680&ds=true&dsyoff=10px&dsblur=42px&wc=true&wa=true&pv=67px&ph=83px&ln=false&fl=1&fm=Space+Mono&fs=14px&lh=220%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522avatar%2520avatar-lg%2520avatar-text%2522%2520alt%253D%2522avatar%2522%253ERM%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2520avatar-md%2520avatar-text%2522%2520alt%253D%2522avatar%2522%253EUI%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2520avatar-sm%2520avatar-text%2522%2520alt%253D%2522avatar%2522%253EYZ%253C%252Fdiv%253E%250A%253Cdiv%2520class%253D%2522avatar%2520avatar-xs%2520avatar-text%2522%2520alt%253D%2522avatar%2522%253ECD%253C%252Fdiv%253E"
+              style="
+                width: 100%;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
```

Comment:
Just a suggestion: I know this inline CSS comes from carbon.sh but you can use one class and add there these properties to avoid having inline CSS for iframe.

Reply:
Thank you, will surely add this.